test_e2e/src/databricks-volumes.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+SRC_PATH=$(dirname "$(realpath "$0")")
+SCRIPT_DIR=$(dirname "$SRC_PATH")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=databricks-volumes
+OUTPUT_DIR=$SCRIPT_DIR/structured-output/$OUTPUT_FOLDER_NAME
+WORK_DIR=$SCRIPT_DIR/workdir/$OUTPUT_FOLDER_NAME
+DOWNLOAD_DIR=$SCRIPT_DIR/download/$OUTPUT_FOLDER_NAME
+CI=${CI:-"false"}
+
+DATABRICKS_VOLUME="test-platform"
+DATABRICKS_VOLUME_PATH="databricks-volumes-test-output-24497"
```

Comment:
Lets find a better document to download. I think it ends up being a .json which I don't think ends up being Partitioned.

 I'll connect with Ronnie and put one up as soon as I can.

Reply:
Ok I modified the `test-databricks-volumes.py` script and with it I uploaded fake-memo to `/Volumes/utic-dev-tech-fixtures/default/test-platform/databricks-volumes-test-output-24497/fake-memo.pdf`